src/state/REPLState.js

```diff
@@ -77,11 +77,11 @@ class REPLState {
   static Decode(encodedState) {
     let jsonState = JSON.parse(encodedState);
     return new REPLState(
-      decodeBase64(jsonState.base64SourceKey),
-      decodeBase64(jsonState.base64PluginKey),
+      decodeBase64(jsonState.base64SourceKey || ""),
+      decodeBase64(jsonState.base64PluginKey || ""),
```

Comment:
![image](https://user-images.githubusercontent.com/22751307/90060720-24a0dc00-dcb3-11ea-9fe0-23d085a819d7.png)

Tried performing the forking procedure, is this supposed to happen (missing the fork icon)?

Reply:
Just checked out master same issue 🙁

Reply:
@penguingovernor should be fixed now after #103 has been merged into `master`